coralproject/talk
# Fix modals to work with shadow DOM

## What does this PR do?

Removes the portal parameter from the Modal implementation since we no longer need it to show modal dialogs within the stream embed.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

- head to stream
- login
- go to the `My Profile` > `Delete my account` and click `Request`
- see that the modal displays correctly
- head to the admin
- check community, stories, and mod queue tabs to open various drawers and modals
- see that the modals display correctly
 
## How do we deploy this PR?

No special considerations.
